packages/jest-diff/yarn.lock

```diff
@@ -1,45 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
-# yarn lockfile v1
-
-
-ansi-regex@^2.0.0:
-  version "2.1.1"
-  resolved "https://registry.yarnpkg.com/ansi-regex/-/ansi-regex-2.1.1.tgz#c3b33ab5ee360d86e0e628f0468ae7ef27d654df"
-
-ansi-styles@^2.2.1:
```

Comment:
This is the reason why the ES5 build fails as `jest-diff` used to use `ansi-styles@2.x` and now is uses whatever the global `jest` package.json defines (which is `ansi-styles@3.0.0`)... 

Why don't we want these lock files anymore? This might cause other issues as all packages dependencies might have changed.